### PR TITLE
Doctests for some filesystem methods

### PR DESCRIPTION
**Pull request by kshyatt**
Also fixed some weird indenting issue?

### CODE REVIEW

**Review by KristofferC — APPROVED** [review]

File: base/stat.jl [hunk 1]
```diff
@@ -165,26 +165,53 @@ Returns `true` if `path` is a FIFO, `false` otherwise.
...
 """
- ischardev(st::StatStruct) = filemode(st) & 0xf000 == 0x2000
+ischardev(st::StatStruct) = filemode(st) & 0xf000 == 0x2000
 
- """
-     isdir(path) -> Bool
+"""
+    isdir(path) -> Bool
+
+Returns `true` if `path` is a directory, `false` otherwise.
```
Comment (fredrikekre) [inline base/stat.jl]: While you are here: `Returns` -> `Return`

File: base/stat.jl [hunk 2]
```diff
@@ -165,26 +165,53 @@ Returns `true` if `path` is a FIFO, `false` otherwise.
...
- Returns `true` if `path` is a block device, `false` otherwise.
- """
+Returns `true` if `path` is a block device, `false` otherwise.
+"""
 isblockdev(st::StatStruct) = filemode(st) & 0xf000 == 0x6000
 
 """
     isfile(path) -> Bool
 
 Returns `true` if `path` is a regular file, `false` otherwise.
```
Comment (fredrikekre) [inline base/stat.jl]: and here.

File: base/stat.jl [hunk 3]
```diff
@@ -165,26 +165,53 @@ Returns `true` if `path` is a FIFO, `false` otherwise.
...
+    isdir(st::StatStruct) = filemode(st) & 0xf000 == 0x4000
 
- """
-     isblockdev(path) -> Bool
+"""
+    isblockdev(path) -> Bool
 
- Returns `true` if `path` is a block device, `false` otherwise.
- """
+Returns `true` if `path` is a block device, `false` otherwise.
```
Comment (fredrikekre) [inline base/stat.jl]: same here

**Review by fredrikekre — APPROVED** [review]
Feel free to ignore my comments

**Comment by kshyatt** [comment]
I think the Travis fail is spurious. I'm going to make another PR to deal with `Returns`.

**Pull request merged**